unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/States/EditorMode/Voxels/VoxelController.cs

```diff
@@ -101,20 +103,11 @@ public void SetEditObjectLikeVoxel()
         }
     }
 
-    public void SetEditionGO(GameObject _editionGO)
-    {
-        editionGO = _editionGO;
-    }
+    public void SetEditionGO(GameObject _editionGO) { editionGO = _editionGO; }
```

Comment:
In order to follow our code conventions...
```suggestion
    public void SetEditionGO(GameObject editionGO) { this.editionGO = editionGO; }
```